unity-renderer/ProjectSettings/Physics2DSettings.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!19 &1
 Physics2DSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 4
+  serializedVersion: 5
```

Comment:
I am not sure we need to push this one. But maybe it is fine and came with Unity Update